docs/_data/mesheryctlcommands/cmds.yml

```diff
@@ -130,14 +153,11 @@ system:
     logs:
       name: logs
       description: Starts tailing Meshery server debug logs
-      usage: |
-
-          # View the logs for a specific component
```

Comment:
This update loses the fact that users can filter logs for a specified component.